docs/reference/indices/aliases.asciidoc

```diff
@@ -168,3 +174,38 @@ stream aliases don't support this parameter.
 Only the `add` action supports this parameter.
 =====
 ====
+
+
+
+[role="child_attributes"]
+[[indices-aliases-api-response-body]]
+==== {api-response-body-title}
+
+`acknowledged`::
+(Boolean)
+If `true`, the request received a response from the master node within the
+`timeout` period.
+
+`errors`::
+(Boolean)
+If `true`, at least one of the requested actions failed.
+
+`action_results`::
+(array of objects) Results for each requested action.
++
+.Properties of `action_results` objects
+[%collapsible%open]
+====
+
+`action`::
+(string)
+The type of the associated action, the values will be one of `add`, `remove`, or `remove_index`.
+
+`success`::
+(boolean)
+If true, the requested action was successfully applied at least once.
```

Comment:
At least once? We potentially run the actions they ask for repeatedly?

Reply:
Kind of. Since the remove action can have wildcards, it can apply to multiple aliases. It's treated as a success if one or more aliases match. Maybe it would be clearer to say: `the requested action was successfully applied to one or more aliases`